packages/terra-heading/tests/nightwatch/heading-spec.js

```diff
@@ -0,0 +1,58 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
My thoughts are testing the CSS implementation details is similar to testing a private implementation of a component. I'd prefer to just test the public API (props) of the component. I'm open to ideas on this.